webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+    print: './src/print.js',
+  },
```

Comment:
- [ ] Webpack is trying to load this `./src/print.js` which is not existing inside the src directory and it's giving an error on page load. Kindly remove this line from your config so the app runs fine with no errors. 👍🏽

<img width="1513" alt="Screenshot 2023-03-27 at 5 14 49 PM" src="https://user-images.githubusercontent.com/50941074/228016686-289ba36c-3dae-4a59-a0c8-bb4864e76667.png">

Reply:
I did thank you bro